use_cases/examples/home_loan_delinquency/deploy/run_dag.py

```diff
@@ -8,7 +8,7 @@
 from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import KubernetesPodOperator
 
 # Project and region for the repository
-PROJECT_ID = Variable.get("PROJECT_ID")
+PROJECT_ID = Variable.get("PROJECT_ID")  # set as env variable in TF as per b/235298528
```

Comment:
Should we leave the comment in there? This is purely internal and doesn't add much. 

This link (from the bug) is descriptive:
https://airflow.apache.org/docs/apache-airflow/stable/howto/variable.html?highlight=variable#storing-variables-in-environment-variables

Reply:
no, we can remove that - I thought to leave it there for now so you could see the documentation explaining the rationale to the changes in the environment variables names